### PR TITLE
HSC-186: Add New Maxillofacial Surgical Procedures

### PR DESCRIPTION
**Pull request by nuhuhmutebi**

### CODE REVIEW

**Review by rbuisson — APPROVED** [review]

File: configuration/concepts/orders_maxillofacial.csv [hunk 1]
```diff
@@ -9,4 +9,21 @@ ab404219-1963-4041-bffb-2d609409523d,,,Exérèse Tumeur de la face,Exérèse Tum
...
+15c2b536-8923-47fa-9486-3b36c4f73cd5,,,Chirurgie périapicale,Chirurgie périapicale,Chirurgie périapicale,Chirurgie périapicale,,,Maxillofacial,Text,,,,,,,,,,,
+ad1e186b-59e0-44de-8d38-6bbc95df6f13,,,Chirurgie préprothétique,Chirurgie préprothétique,Chirurgie préprothétique,Chirurgie préprothétique,,,Maxillofacial,Text,,,,,,,,,,,
+8fbe3366-9b61-4782-929e-e922c7a7bdb5,,,Fracture de Lefort I,Fracture de Lefort I,Fracture de Lefort I,Fracture de Lefort I,,,Maxillofacial,Text,,,,,,,,,,,
+7bebe672-17ef-4dc9-8998-4519a738a980,,,Fracture de Lefort II,Fracture de Lefort II,Fracture de Lefort II,Fracture de Lefort II,,,Maxillofacial,Text,,,,,,,,,,,
+941f4b5b-74a1-4d98-aaea-b581cb2ac973,,,Fracture de Lefort III,Fracture de Lefort III,Fracture de Lefort III,Fracture de Lefort III,,,Maxillofacial,Text,,,,,,,,,,,
+88333bde-fe5f-4d5e-8e2a-1775e7d14166,,,Fracture Mandibulaire,Fracture Mandibulaire,Fracture Mandibulaire,Fracture Mandibulaire,,,Maxillofacial,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/orders_maxillofacial.csv]: Fracture mandibulaire

File: configuration/concepts/orders_maxillofacial.csv [hunk 2]
```diff
@@ -9,4 +9,21 @@ ab404219-1963-4041-bffb-2d609409523d,,,Exérèse Tumeur de la face,Exérèse Tum
...
+d194eaea-f26a-4e2b-9384-03f4b5057b2c,,,Fracture nasale,Fracture nasale,Fracture nasale,Fracture nasale,,,Maxillofacial,Text,,,,,,,,,,,
+aa50f047-f4a5-4a88-9f96-c8a7c808420d,,,Fracture Zygomatique,Fracture Zygomatique,Fracture Zygomatique,Fracture Zygomatique,,,Maxillofacial,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/orders_maxillofacial.csv]: Fracture zygomatique

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: configuration/concepts/orders_maxillofacial.csv (hunk 2 above)
Comment (nuhuhmutebi) [reply configuration/concepts/orders_maxillofacial.csv]: Done

File: configuration/concepts/orders_maxillofacial.csv (hunk 1 above)
Comment (nuhuhmutebi) [reply configuration/concepts/orders_maxillofacial.csv]: Done

**Pull request merged**